packages/wallet-ui/src/store/types.ts

```diff
@@ -13,8 +13,8 @@ export type User = {
 };
 
 export type AssetType = {
-  name: string;
-  shortName: string;
+  name?: string;
```

Comment:
Is there a case where this and `shortName` won't be available? :thinking:

Reply:
yes, because the calls to the list of currencies of the network and the calls to get various balances are done at different times, and not always have the entire information.

For example, a getIndexedBalances doesn't have the name and shortName, but I saved that anyway as an AssetType, because it's expected that at some point, that gets transformed into an assetType, with all the necessary data.